3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala

```diff
@@ -97,7 +90,7 @@ object EnrichJobSpec {
     def apply[S <: String](actual: Expectable[S]) = {
       result((unmatcheable && expected == null) || actual.value == expected,
         "%s (index: %s): %s ".format(field, index, if (unmatcheable) "is unmatcheable" else "%s equals %s".format(actual.description, expected)),
-        "%s (index: %s): %s does not equal %s".format(field, index,  actual.description, expected),
+        "%s (index: %s): %s does not equal %s".format(field, index, actual.description, expected),
```

Comment:
I know it wasn't added in this commit, but can we change `format` to string-interpolation.

Reply:
I have no problem changing it. @BenFradet The original (458f2e8cf0) commit uses format instead fo string interpolation. Any reason to prefer format here?

Reply:
this is just inherited from the scalding code, I don't have any reason

Reply:
Done